src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -183,7 +196,33 @@ def get(self, request, organization):
 
         results = list(cursor_result)
 
-        context = serialize(results, request.user, serializer())
+        snuba_filters = []
+        if "search_filters" in query_kwargs and query_kwargs["search_filters"] is not None:
+            snuba_filters = [
+                convert_search_filter_to_snuba_query(search_filter)
+                for search_filter in query_kwargs["search_filters"]
+                if search_filter.key.name not in self.skip_snuba_fields
+            ]
+
+        lifetime_stats = serialize(results, request.user, serializer())
```

Comment:
Won't this serialize more than just the stats and then throw away most of that work? Should we extract the stats generation so that we don't waste work?

Reply:
@fpacifici Also had comments relating to these queries.

My suggestion is to release this version internally for testing and improve the queries for general releae